p2panda-rs/src/operation/plain/plain_fields.rs

```diff
@@ -191,7 +196,7 @@ mod tests {
             ("it_works", PlainValue::Boolean(true)),
             (
                 "it_works",
-                PlainValue::StringOrRelation("... and ignores duplicates".into()),
+                PlainValue::ByteString(ByteBuf::from("... and ignores duplicates")),
```

Comment:
I wonder if the name isn't confusing, I read it as "it is a utf8 string in its byte representation" which is half true? Maybe just calling it "Bytes" as well or "AmbiguousBytes"? 🥴